SwiftPackageTests/Shared/SharedTests.swift

```diff
@@ -0,0 +1,27 @@
+import XCTest
+
+class SharedTests: XCTestCase {
+
+  override func setUp() {
+    super.setUp()
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+  }
+
+  override func tearDown() {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    super.tearDown()
+  }
+
+  func testExample() {
+    // This is an example of a functional test case.
+    // Use XCTAssert and related functions to verify your tests produce the correct results.
+  }
+
+  func testPerformanceExample() {
+    // This is an example of a performance test case.
+    self.measure {
+      // Put the code you want to measure the time of here.
+    }
+  }
+
```

Comment:
@zenangst can Marvin on Xcode 8 can still automatically add this new line at the end?